plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/AdvancedConnectionEditor.java

```diff
@@ -308,13 +308,12 @@ private void discoverRegistryPressed(Shell shell) {
 				// Verify with user
 				String title = "Overwrite registry URL?";
 				String msg = "Are you sure you want to change the registry URL from " + oldVal + " to " + newVal + "?";
-				MessageBox mb = new MessageBox(shell, SWT.OK | SWT.CANCEL);
-				mb.setText(title);
-				mb.setMessage(msg);
+				MessageDialog dialog = new MessageDialog(shell, title, null, 
+						msg, MessageDialog.CONFIRM, new String[] {"Cancel", "OK"} , 0);
```

Comment:
this leads to a dialog that has the order of "OK" and "Cancel" inversed (most dialogs in Eclipse have the opposite order 1. Cancel 2.OK):
![image](https://user-images.githubusercontent.com/25126/32831951-3bb1667c-c9fa-11e7-8c42-f301f9d580fb.png)